oap-server/server-starter/src/main/resources/otel-rules/clickhouse/clickhouse-service.yaml

```diff
@@ -0,0 +1,155 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+filter: "{ tags -> tags.job_name == 'clickhouse-monitoring' }" # The OpenTelemetry job name
+expSuffix:  tag({tags -> tags.host_name = 'clickhouse::' + tags.host_name}).service(['host_name'] , Layer.CLICKHOUSE)
+metricPrefix: meter_clickhouse
+metricsRules:
+  # Network
+  # Number of alive connections.
+  - name: connections_alive
+    exp: ClickHouseMetrics_KeeperAliveConnections.sum(['host_name'])
```

Comment:
About this workflow, I want to ask, is every clickhouse node exposing an endpoint?
If it's so, these metrics(service level) seem not correct. Let's say there are two Clickhouse nodes, each node has its the number of connections, OTEL will scrap two places and forward to OAP, then, without specific downsampling process, in one minute, the data from each node would overlap and override each other. How is this processed now?

Reply:
In this case, if there are two nodes

In the case A
ck node1 -> OTEL -> OAP
ck node2-> OTEL -> OAP

In the case B
T1, ck node1 -> OTEL -> OAP
T2, ck node1 -> OTEL -> OAP
(T1 T2) are in the same minute(same for in hour/day)

Then from the OAP process perspective, it can't tell A and B, so, if you work on downsampling(default as AVG), it could be AVG of node1 and node2(case A), or AVG of ck node1 from T1 and T2(case B). 
For more complex case, it could a combination of case A and B, in one minute, N scrapings from node1 and M scrapings from node2, then the default sampling is doing AVG for N + M scrapings from node1 and node2. This actually is not the value of the aggregated metric for the whole service(multiple nodes).


Could you explain more about this case about this?

Reply:
It doesn't make sense to present like `AVG for N + M scrapings from node1 and node2`, I will aggregate the metrics by MQE, so do the `Zookeeper Sessions & ZooKeeper Watch`, right?

Reply:
I wonder what kind of metric have to  be the  aggregated value, and what kind of metric can be the AVG value.
In this case, the `alive connection/file open/zookeeper session/zookeeper watch` shown in `CARD` have to be accurate, what about other metrics in `Line` or `Area`.

Reply:
I think it doesn't rely on the widget type. It is more about the meaning of the metrics.
A metric value is from each server, so the service level metric always has to be a kind of aggregated for all servers(instances). 

Why do you think `card` widget different from `Line` and `Area`?

Reply:
```
# HELP ClickHouseMetrics_ZooKeeperSession Number of sessions (connections) to ZooKeeper. Should be no more than one, because using more than one connection to ZooKeeper may lead to bugs due to lack of linearizability (stale reads) that ZooKeeper consistency model allows.
# TYPE ClickHouseMetrics_ZooKeeperSession gauge
ClickHouseMetrics_ZooKeeperSession{host_name="clickhouse:8123",instance="clickhouse1:9363",job="clickhouse-monitoring"} 1
```
The MAL processes the source metrics in time before storage, actually your OTEL collects 2 sources and sends metrics to OAP separately (clickhouse1,clickhouse2), you can see the above source metric only includes `clickhouse1`. So you can't sum `host_name` in MAL, if you do that the sum result will only include `clickhouse1 or clickhouse2`. 
I suggest the expression in service metric:
`ClickHouseMetrics_ZooKeeperSession.sum(['host_name'，service_instance_id])`
Then we can keep the instance as a label to show 2 lines on the widget.
Or you can use MQE(processes the metric from storage) func `aggregate_labels` to sum the result into a single value.